.github/workflows/rebuild-readme-notebooks-table-on-pr-merge-to-main.yml

```diff
@@ -0,0 +1,29 @@
+name: rebuild readme.md notebooks table on pr merge to main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Rebuild notebooks table in project README.md
+        run: python automation/autogenerate_notebooks_table.py
+      - name: Check for modified files
+        id: git-check
+        run: echo ::set-output name=modified::$(if [ -n "$(git status --porcelain)" ]; then echo "true"; else echo "false"; fi)
+      - name: Update changes in GitHub repository
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          git config --global user.name 'gpintoruiz'
+          git config --global user.email 'gpintoruiz@users.noreply.github.com'
```

Comment:
i think maybe is better to do this automatic commit on behalf of a bot, or your profile may be filled with garbage of commits

maybe with:
```
    git config --local user.email "github-actions[bot]@users.noreply.github.com"
    git config --local user.name "GitHub Actions [Bot]"
```

Reply:
Did not know it existed 😅, updated